src/main/scala/is/hail/expr/Type.scala

```diff
@@ -642,6 +642,14 @@ final case class TAggregable(elementType: Type) extends TContainer {
     throw new RuntimeException("TAggregable is not realizable")
 }
 
+object TContainer {
+  def loadLength(region: MemoryBuffer, aoff: Long): Int =
+    region.loadInt(aoff)
+
+  def loadLength(region: Code[MemoryBuffer], aoff: Code[Long]): Code[Int] =
+    region.invoke[Long, Int]("loadInt", aoff)
```

Comment:
You can just write `region.loadInt(aoff)`, right?

Reply:
ah yep.